geomesa-tools/bin/geomesa-common.sh

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Apache License, Version 2.0 which
+# accompanies this distribution and is available at
+# http://www.opensource.org/licenses/apache2.0.php.
+#
+
+# Common functions for GeoMesa command line tools
+
+# Set environment variables in bin/geomesa-env.sh
+# Load config Env Variables
+. "${0%/*}"/geomesa-env.sh
+
+function setGeoHome() {
+    SOURCE="${BASH_SOURCE[0]}"
+    # resolve $SOURCE until the file is no longer a symlink
+    while [ -h "${SOURCE}" ]; do
+        bin="$( cd -P "$( dirname "${SOURCE}" )" && pwd )"
+        SOURCE="$(readlink "${SOURCE}")"
+        # if $SOURCE was a relative symlink, we need to resolve it relative to the path where
+        # the symlink file was located
+        [[ "${SOURCE}" != /* ]] && SOURCE="${bin}/${SOURCE}"
+    done
+    bin="$( cd -P "$( dirname "${SOURCE}" )" && cd ../ && pwd )"
+    export GEOMESA_HOME="$bin"
+    export PATH=${GEOMESA_HOME}/bin:$PATH
+    echo >&2 "Warning: GEOMESA_HOME is not set, using $GEOMESA_HOME"
```

Comment:
can we print this to the geomesa log instead of std out...i'm really tired of seeing it be honest :)